cookbook/entities/deleting_tasks.rst

```diff
@@ -0,0 +1,113 @@
+How to Delete Tasks
+===================
+
+The data grid that you created so far is able to let the user delete entries without having to
+reload the entire page. To achieve this you have to follow these three steps:
+
+#. :ref:`Configure an API manager service <cookbook-entity-api-manager-service>`.
+
+#. :ref:`Create a controller that deletes a task <cookbook-entity-delete-controller>`.
+
+#. :ref:`Link to the controller from the data grid <cookbook-entity-delete-grid-config>`.
+
+.. _cookbook-entity-api-manager-service:
+
+Configure the API Manager Service
+---------------------------------
+
+Fortunately, the OroSoapBundle comes with the ``oro_soap.manager.entity_manager.abstract`` service
```

Comment:
Please change this sentence. Not only this service can delete Doctrine's entities.